client/packages/common/src/intl/locales/en/common.json

```diff
@@ -181,16 +181,17 @@
   "dashboard": "Dashboard",
   "description.already-issued": "The number of units already issued in shipments.",
   "description.available-months-of-stock": "Available months of stock on hand",
-  "description.available-soh": "Available Stock on Hand",
+  "description.available-soh": "Available Stock on Hand.  $t(description.doses-quantity)",
```

Comment:
Side quest, request from Mark, to change from `dos.` to `ds` and explain in the column descriptions